dbt/include/glue/macros/adapters.sql

```diff
@@ -128,37 +30,12 @@
   {%- endif %}
 {%- endmacro -%}
 
-{% macro glue__create_view_as(relation, sql) -%}
-    {{ adapter.create_view_as(relation, sql) }}
-{% endmacro %}
-
-{% macro glue__create_view(relation, sql) -%}
-    {{ adapter.create_view_as(relation, sql) }}
-{% endmacro %}
 
 {% macro glue__snapshot_get_time() -%}
   datetime()
 {%- endmacro %}
 
-{% macro glue__rename_relation(from_relation, to_relation) -%}
-    {{ adapter.rename_relation(from_relation, to_relation) }}
-{% endmacro %}
-
-{% macro glue__make_temp_relation(base_relation, suffix) %}
-    {% set tmp_identifier = base_relation.identifier + '_tmp' ~ suffix %}
```

Comment:
AFAIK, `suffix` is the `tmp` part, you don't also need `_tmp`. If that is the case, then you don't need this macro. But maybe there's a good reason? idk